pages/blog.module.scss

```diff
@@ -0,0 +1,110 @@
+@import '../styles/variables';
+@import '../styles/mixins';
+
+.featured {
+  background: #000;
+  color: #fff;
+  margin-top: $header-height;
+}
+
+.secondary {
+  background: #242424;
+  color: #fff;
+}
+
+.featured {
+  @include breakpoint(medium-up) {
+    margin: 0 -3.5rem;
+    padding: 7.25rem 18rem;
+  }
+
+  @include breakpoint(medium-only) {
+    padding: 6rem 5rem;
+  }
+
+  @include breakpoint(small) {
+    padding: 0 1rem 1.5rem;
+    text-align: start;
+  }
+}
+
+.secondary,
+.allBlogs {
+  @include breakpoint(medium-up) {
+    margin: 0 -3.5rem;
+    padding: 5rem 18rem;
+  }
+
+  @include breakpoint(medium-only) {
+    padding: 5rem 5rem;
+  }
+
+  @include breakpoint(small) {
+    padding: 3rem 1rem 1rem;
+    text-align: start;
+  }
+}
+
+.secondaryTitle {
+  @include h3;
+  margin: 0 0;
+  font-weight: 500;
```

Comment:
In the design it looks like the font size isn't exactly the same as our `@include h3`, which is a little frustrating.

Their spec is this:

```css
font-size: 24px;
line-height: 32px;
```